src/Utilities/DiagnosticCategory.cs

```diff
@@ -13,7 +13,21 @@ internal static class DiagnosticCategory
         public static readonly string Security = AnalyzerUtilitiesResources.CategorySecurity;
         public static readonly string Usage = AnalyzerUtilitiesResources.CategoryUsage;
         public static readonly string Naming = AnalyzerUtilitiesResources.CategoryNaming;
-        public static readonly string ApiDesign = AnalyzerUtilitiesResources.CategoryApiDesign;
         public static readonly string Library = AnalyzerUtilitiesResources.CategoryLibrary;
+        public static readonly string Documentation = AnalyzerUtilitiesResources.CategoryDocumentation;
```

Comment:
Also filed https://github.com/dotnet/roslyn-analyzers/issues/1468 to track why we use resource strings for Category in all our FxCop analyzers, but use string constants in Roslyn and MS.CA analyzers.